src/main/java/FrameworksDrivers/ScheduleGraphics.java

```diff
@@ -79,6 +72,7 @@ public ScheduleGraphics() {
         buttonPanel = new JPanel();
 
         titleLabel = new JLabel("Scheduler App");
+
         mondayLabel = new JLabel("Monday");
         tuesdayLabel = new JLabel("Tuesday");
         wednesdayLabel = new JLabel("Wednesday");
```

Comment:
Sorry if this is obvious or dumb but why is there no Thursday and Friday label?

Reply:
i think it was automatically hidden by github because there weren't any changes made to it -- there's like blue arrow icons on the side you can click to expand and view the hidden code. i can see the thurs and fri labels when i do that